src/styles/stars.scss

```diff
@@ -0,0 +1,41 @@
+@import "./utils/variables";
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 16px;
+
+  &__list {
+    display: flex;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+
+    &-item {
+      background-image: url("../images/star.svg");
+      width: 16px;
+      height: 16px;
+      background-repeat: no-repeat;
+      background-position: center;
+    }
+
+    &-item:not(:last-child) {
+      margin-right: 4px;
+    }
+
+    &-item:nth-child(-n + 4) {
```

Comment:
apply star styles from a previous task. it should cover all modifier cases (1-5)